### ISSUE TITLE
Vernacular names for Erythranthe patula

### ISSUE THREAD

**Issue by brouille** [Canadensys/vascan-data#2398]
@MFavreau 
Erythranthe patula has been added to VASCAN for BC
Vernacular names are required
No FNA treatment available but eFlora BC has a treatment (as Mimulus patulus)
Thanks
Luc

**Comment by MFavreau** [comment]
DISTRIBUTION : Nesom (2012) mentions the species from Alberta. I would assume the species should also be mentioned (or excluded) from Alberta in VasCan.

FRENCH VERNACULAR : According to Nesom (2012), the plant is erect to ascending, while other species are distinguished in his key as being more prostrate, so I avoided a direct translation of the epithet « patula », i.e. « spreading », « étalé ». For the French name, I chose t« mimule à longs pétioles », refering to the characteristically long petioles mentionned by Nesom. This partly corresponds to the English name « stalk-leaved monkeyflower » used in e-flora BC.

REFERENCE : Nesom, G.L. 2012, Taxonomy of Erythranthe sect. Mimulosma (Phrymaceae).  Phytoneuron 2012-41: 1–36. http://www.phytoneuron.net/PhytoN-sectMimulosma.pdf
